platform/android/MapboxGLAndroidSDKTestApp/build.gradle

```diff
@@ -9,7 +9,7 @@ android {
     compileSdkVersion androidVersions.compileSdkVersion
 
     defaultConfig {
-        applicationId "com.mapbox.mapboxsdk.testapp"
+        applicationId "org.maplibre.testapp"
```

Comment:
@fynngodau Is this correct?

Reply:
Yes, the package name can be arbitrary (we talked about this on Slackl.

Reply:
Ah I remember.